src/components/node.js

```diff
@@ -236,9 +236,10 @@ class Node extends React.Component<INodeProps, INodeState> {
       // mousedown via dragging (see graph-view asyncElevateNodeAndEdges)
       const target = e.target;
       const options = Object.assign(e);
+      const eventType = e.detail >= 2 ? 'dblclick' : 'click';
```

Comment:
why >= not === ?

Reply:
From the [info page](https://developer.mozilla.org/en-US/docs/Web/API/UIEvent/detail), it says for "For `mousedown` or `mouseup` events, UIEvent.detail is 1 plus the current click count.". When I was testing it, I didn't see it return `> 2`, but since `handleDragEnd` is triggered on `onMouseUp`, I did `>=` to be on the safe side